pycbc/psd/variation.py

```diff
@@ -113,10 +113,10 @@ def calc_filt_psd_variation(strain, segment, short_segment, psd_long_segment,
     # Convert start and end times immediately to floats
     start_time = float(strain.start_time)
     end_time = float(strain.end_time)
+    srate = int(sample_rate)
 
     # Resample the data
-    strain = resample_to_delta_t(strain, 1.0 / 2048)
-    srate = int(strain.sample_rate)
+    strain = resample_to_delta_t(strain, 1. / srate)
```

Comment:
Can I check: wouldn't `strain` here already be at the `opt.sample_rate`? What sample rate would it have if not `opt.sample_rate`? Do I not achieve the same thing by just removing the `resample_to_delta_t` line?

@Moz-Port has left academia now, but I'm not aware of any reason why this was resampled specifically to 2048Hz ... I hope there's not a particular reason why this needs doing that we're missing ???

Reply:
Oh yes! We don't need to resample it again!